app/templates/forum.html

```diff
@@ -1,14 +1,37 @@
-﻿{% extends "base.html" %}
-{% block style %}
-<link rel="stylesheet" href="../static/content/dashboardstyle.css">
+﻿{% block style %}
+<link rel="stylesheet" href="{{ url_for('static', filename='content/forumstyle.css') }}">
+<link href="//netdna.bootstrapcdn.com/font-awesome/3.2.1/css/font-awesome.css" rel="stylesheet">
 {% endblock %}
 {% block content %}
-<div>
-    <a href="{{ url_for('create_thread',groupID=groupID) }}"> Create Thread</a> <br />
+{% include "sidebar.html" %}
+<main class="forum-grid">
+    <div class="forum-header">
+        <h1>{{ group.name }} FORUM</h1>
+        <a href="{{ url_for('create_thread',groupID=groupID) }}"><h5><i class="fas fa-plus"> Create A New Thread</i></h5></a>
+    </div>
+    <div class="forum-title">
+        <h2>Post Title</h2>
+    </div>
+    <div class="forum-count">
+        <h2>Number of Posts</h2>
+    </div>
+    <div class="forum-time">
+        <h2>Last Post</h2>
+    </div>
     {% for thread in threads %}
-    <div>
-        <a href="{{ url_for('forum_post', groupID=groupID, threadID=thread.id) }} ">{{thread.title}}</a>
+    <div class="post-title">
+        <a href="{{ url_for('forum_post', groupID=groupID, threadID=thread.id) }} ">
+            <h1>{{thread.title}}</h1>
+            <h4>Thread content maybe?</h4>
```

Comment:
`Thread` content is basically the first `Post`'s content. `Thread` itself has no content.

Reply:
How would we access a snippet of the post content? {{ post.content }}?

Reply:
Yep correct. We only see the `Post.content` inside `posts.html`.